src/main/resources/assets/buildinggadgets/lang/de_de.lang

```diff
@@ -1,43 +1,89 @@
 message.gadget.invalidblock=Block ungültig
+message.gadget.nothingtoundo=Nichts zum Rückgängigmachen
+message.gadget.pastefailed=Einfügen fehlgeschlagen, ungültiges JSON
+message.gadget.pastesuccess=Einfügen aus Zwischenablage erfolgreich
+message.gadget.pastetoobig=Einzufügende Daten sind zu groß
+message.gadget.rotated=Blöcke rotiert
+message.gadget.toobigarea=Bereich zu groß, maximaler Bereich ist 125x125x125 (Rechtsklicke in die Luft zum Zurücksetzen)
 message.gadget.toolmode=Werkzeugmodus
 message.gadget.toolrange=Werkzeugreichweite
-message.gadget.anchorrender=Darstellung festgesetzt
-message.gadget.anchorremove=Festsetzen aufgehoben
-message.gadget.nothingtoundo=Nichts zum Rückgängigmachen
+message.gadget.toomanyblocks=Zu viele Blöcke, Limit ist 32768 (Rechtsklicke in die Luft zum Zurücksetzen)
 message.gadget.undofailed=Fehler beim Rückgängigmachen (zu weit weg?)
-message.gadget.fuzzymode=Ungenauerer Modus
-message.gadget.copied=Bereich kopiert
-item.buildinggadgets.constructionpastecontainer.name=Pastenbehälter
-item.buildinggadgets.constructionpaste.name=Baupaste
-tooltip.pasteContainer.amount=Menge
-block.buildinggadgets.constructionblock.name=Baustoffblock
-tile.buildinggadgets.constructionblock.name=Baustoffblock
-block.buildinggadgets.constructionblockpowder.name=Baustoffpulver
-tile.buildinggadgets.constructionblockpowder.name=Baustoffpulver
-tile.buildinggadgets.templatemanager.name=Schablonen-Manager
-block.buildinggadgets.templatemanager.name=Schablonen-Manager
-tooltip.constructionblockpowder.helptext=Neben Wasser zu platzieren
-item.buildinggadgets.copypastetool.name=Kopier-Gadget
-message.gadget.rotated=Blöcke rotiert
-message.gadget.toomanyblocks=Zu viele Blöcke, Limit ist 32768 (Rechtsklick in die Luft zum Zurücksetzen)
-message.gadget.toobigarea=Bereich zu groß, Maximalgröße ist 125x125x125 (Rechtsklick in die Luft zum Zurücksetzen)
-message.gadget.areareset=Kopierbereich zurückgesetzt
-tooltip.template.name=Name
-item.buildinggadgets.template.name=Schablone
-message.gadget.pastetoobig=Eingefügtes ist zu groß
-message.gadget.pastefailed=Einfügen fehlgeschlagen: Ungültiges JSON
-message.gadget.copyfailed=Kopieren fehlgeschlagen: Keine Daten auf dem Gadget
-message.gadget.copyguierror="Bereichkopie fehlgeschlagen"
+
+#help: 
+help.gui.buildinggadgets.area.arrow.data_flow=Stellt den Datenfluss von einem Item auf ein anderes dar, entweder von Schablone zu Gadget oder einer anderen Schablone, oder von Gadget/Schablone auf eine Schablone oder ein Stück Papier.
+help.gui.buildinggadgets.area.field.template_name=Wenn Daten gespeichert oder eingefügt werden, wird die kreierte Schablone diesen Namen haben.
+help.gui.buildinggadgets.area.preview=Vorschau der in Gadget oder Schablone gespeicherten Daten
+help.gui.buildinggadgets.area.slot.gadget=§nKopier-und-Einfüge-Gadgets§r oder §nSchablonen§r können von hier aus eingelesen oder beschrieben werden.\n\nEin §9farbiges Overlay§r zeigt an, wenn das Item Daten erhalten wird.
```

Comment:
```suggestion
 help.gui.buildinggadgets.area.slot.gadget=§nKopier-und-Einfüge-Gadgets§r oder §nSchablonen§r können von hier aus eingelesen oder beschrieben werden.\n\nEin §9farbiges Overlay§r zeigt an, dass das Item Daten erhalten wird.
```
Folge statt Bedingung (it indicates **that** it will receive Data nicht **if**)

Reply:
If you want to say that an Overlay will be shown if the Item recieves Data, then it should be something along the lines of:
```suggestion
 help.gui.buildinggadgets.area.slot.gadget=§nKopier-und-Einfüge-Gadgets§r oder §nSchablonen§r können von hier aus eingelesen oder beschrieben werden.\n\nEin §9farbiges Overlay§r wird angezeigt, wenn das Item Daten erhält.
```